homepages: replace people.engr.ncsu.edu

osalbahr opened the pull request:
[This website was decommissioned on December 15, 2022](https://web.archive.org/web/20230507120755/https://people.engr.ncsu.edu/).

# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (please don't contact us for at least three months for updates, unless you are a sponsor - see below).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR PULL REQUEST WILL BE SUMMARILY REJECTED.
You must read and check **all** the boxes below by filling them in with an X or your PR will be rejected.

**The Basics**

- [x] All pull requests and issues must come from non-anonymous accounts. Make sure your GitHub profile contains your full name.

- [x] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [x] Combine multiple updates to a single institution into a **single PR.**

- [x] Only submit one pull request per institution.

- [x] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [x] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [x] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [x] Check to make sure that you have no spaces after commas, or any missing fields.

- [x] Check to make sure the home page is correct.

- [x] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [x] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [x] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [x] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

osalbahr committed:
homepages: replace people.engr.ncsu.edu

[This website was decommissioned on December 15, 2022](https://web.archive.org/web/20230507120755/https://people.engr.ncsu.edu/).

osalbahr commented:
@emeryberger should I revert `csrankings.csv`?

emeryberger commented:
>  Insert new faculty in alphabetical order (not at the end) in the appropriate csrankings-[a-z].csv files. Do not modify csrankings.csv, which is auto-generated.

emeryberger commented:
> ERROR: Invalid file modification (csrankings.csv). Please only modify allowed CSV files.
> Error: Process completed with exit code 255.

osalbahr commented:
Ok, I think I did it right this time

emeryberger commented on csrankings-h.csv:
```diff
@@ -966,7 +966,7 @@ Huiping Cao,New Mexico State University,http://www.cs.nmsu.edu/~hcao,M6UHVQwAAAA
 Huisheng Chi,Peking University,http://www.cis.pku.edu.cn/auditory/Staff/Prof.Chi.html,NOSCHOLARPAGE
 Huisi Wu,Shenzhen University,http://csse.szu.edu.cn/staff/~hswu,hUb7j9EAAAAJ
-Huiyang Zhou,North Carolina State University,https://people.engr.ncsu.edu/hzhou,_2gFUN0AAAAJ
+Huiyang Zhou,North Carolina State University,https://ece.ncsu.edu/people/hzhou,_2gFUN0AAAAJ
```
Not in CS. Please clarify how they meet the inclusion criteria.

osalbahr replied:
Not sure, I only fixed the url. It looks like the professor was in the list [from the initial commit](https://github.com/emeryberger/CSrankings/blame/2c47b66859fef84b172fcb016a6b9f8c775a968f/csrankings-h.csv#L983). The professor is active in the website's list of conferences, though. Did the inclusion criteria change since then?

emeryberger commented on csrankings-m.csv:
```diff
@@ -1980,7 +1980,7 @@ Mihai Ordean,University of Birmingham,https://www.birmingham.ac.uk/staff/profile
 Mihai Sanduleanu,Masdar Institute,https://www.masdar.ac.ae/aboutus/management/faculty-directory/item/5662-mihai-sanduleanu,sKP9fu0AAAAJ
 Mihai Surdeanu,University of Arizona,http://surdeanu.info/mihai,a3133-8AAAAJ
-Mihail L. Sichitiu,North Carolina State University,https://people.engr.ncsu.edu/mlsichit,P8BZmsYAAAAJ
+Mihail L. Sichitiu,North Carolina State University,https://ece.ncsu.edu/people/mlsichit,P8BZmsYAAAAJ
```
See above (ECE).

emeryberger commented on csrankings-y.csv:
```diff
@@ -726,7 +726,6 @@ Yuan-Fang Wang,Univ. of California - Santa Barbara,http://www.cs.ucsb.edu/~yfwan
 Yuan-Sen Ting,Australian National University,https://comp.anu.edu.au/people/yuan-sen-ting/,FNvucsoAAAAJ
 Yuan-Shun Dai,UESTC,http://www.scse.uestc.edu.cn:8080/teacherteam-english/teacher/view.html?id=44,DRCdhcwAAAAJ
 Yuanchao Liu,Harbin Institute of Technology,http://homepage.hit.edu.cn/ycliu,NOSCHOLARPAGE
-Yuanchao Xu 0001,Univ. of California - Santa Cruz,https://yuanchaoxu6.github.io/,kIvi43wAAAAJ
```
Why this removal?

ryanzhumich committed:
Update for Pennsylvania State University (#6397)

* Add Abhinav Verma

* Add Anton Bankevich

* Add Kiwan Maeng

* Add Ruslan Nikolaev

* Add Wenpeng Yin

* Add Yana Safonova

* Add Young Kun-Ko

danyaljj committed:
CS @ JHU  (#5945)

* Anjalie and Eric.

* Renjie.

* minor correction.

* Adding the remainder of the affiliates.

* minor.

* minor.

* minor.

* minor.

* minor.

* minor.

* Ziang and Anjalie

* Ziang and Anjalie

* adding alex marder

* minor.

* minor.

* add comments to the validation file.

* add comments to the validation file.

* add comments to the validation file.

* add comments to the validation file.

* drop the old comments.

* minor.

---------

Co-authored-by: Emery Berger <emery.berger@gmail.com>

xysLong committed:
Add Mayer, Ernst, Jakobs to LMU Munich; Move Blanchette to LMU Munich (#6056)

* Update csrankings-j.csv (Move Jasmin Blanchette to LMU Munich)

* Update csrankings-s.csv (Add Sven Mayer to LMU Munich)

* Update csrankings-g.csv (Add Gidon Ernst to LMU Munich)

* Update csrankings-m.csv (Add Marie-Christine Jakobs to LMU Munich)

* Update csrankings-m.csv (Directly copy name from DBLP entry)

liyistc committed:
Update csrankings-y.csv (#6058)

Adding "Yi Li 0008" to Nanyang Technological University faculty list.

lilicoding committed:
Update csrankings-l.csv (#6065)

Update Li Li's affiliation

ayzk committed:
Dr. Kai Zhao moved from UAB to FSU (#6072)

jay-tau committed:
Update BITS Pilani-Goa faculty (#6048)

* Update BITS Pilani-Goa faculty.
- DBLP name correction: A. Baskar (from A. Bhaskar), Bharat M. Deshpande (from Bharat Deshpande)
- Google Scholar ID correction: A. Baskar
- New additions: Diptendu Chatterjee, Harikrishnan N.B. (Harikrishnan Nellippallil Balakrishnan), Rajesh Kumar 0012, Sanjay Kumar Sahay, Santonu Sarkar
- Moved to industry: Anup Basil Mathew, Pritam Bhattacharya, Soumyadip Bandyopadhyay (Soumyadip Banerjee 0001)
- Updated affiliation: Abhishek Kr Singh (Tel Aviv University), Tirtharaj Dash (University of California - San Diego)
- Removal: Raj K. Jaiswal (no longer at BITS Goa)
- Update homepage links for multiple faculty.
- Add alternative name for Arnab Kumar Paul.

* Update Sougata Sen's homepage

* Switch homepage to HTTPS

* Add Devashish Gosain

* Remove Rizwan

---------

Co-authored-by: Emery Berger <emery.berger@gmail.com>

gsarti committed:
Updating NLP faculty at University of Groningen (#6079)

* Updating NLP faculty at University of Groningen

* Fix names

actbee committed:
Add faculties from Hunan University (#6088)

* added faculties from Hunan University

* fixed

* updated

* updated

* updated

kiwih committed:
Add Hammond Pearce (new lecturer) from UNSW (#6091)

* Add new lecturer from UNSW

* Add second DBLP name for Hammond Pearce

dpfried committed:
Add new faculty from Carnegie Mellon LTI (https://www.lti.cs.cmu.edu/directory/all/154/1) starting Aug 2023 (#6096)

* Update csrankings-d.csv

* Update csrankings-c.csv

* Update csrankings-f.csv

* Update csrankings-l.csv

* Update csrankings-f.csv

* Update csrankings-m.csv

nunoplopes committed:
Add 4 PL professors from IST/U Lisbon (#6109)

* Add 4 PL professors from IST/U Lisbon

* Added missing alias

---------

Co-authored-by: Emery Berger <emery.berger@gmail.com>

mod20388 committed:
Add GSSI faculty (#6114)

CS-BruceChen committed:
add new faculty of Wuhan University into csranking-s.csv and fixed ho… (#6115)

* add new faculty of Wuhan University into csranking-s.csv and fixed homepage error

* Removed https to fix web page

---------

Co-authored-by: Emery Berger <emery.berger@gmail.com>

yalongyang committed:
Update Yalong Yang's affiliation. (#6116)

Yalong Yalong is joining GaTech 2023 Fall.

psoni55 committed:
added Pratik Soni as a faculty of University of Utah (#6117)

* added Pratik Soni as a faculty

* up

* corrected syntax issues with webpage

* updated webpage link

adamperer committed:
adding Adam Perer to CMU (#6118)

emeryberger committed:
Added URI.

swallace21 committed:
add all university of rhode island cs profs (#6120)

* add tenure-track cs profs from the university of rhode island

* add google scholar pages

* match dblp names

* Update csrankings-n.csv

Change Noah Daniels' google scholar to NOSCHOLARPAGE

* Update csrankings-k.csv

fix Krishan's old homepage

---------

Co-authored-by: Emery Berger <emery.berger@gmail.com>

DeqingFu committed:
Add Matteo Sesia of USC (#6128)

* add matteo

* remove space

* add google scholar id

youngwoon committed:
Add 8 faculty members to Yonsei University (#6134)

* Add Dongha Lee to Yonsei University

* Add Jongduk Back to Yonsei University

* Move Seong Jae Hwang to Yonsei University

* Add Youngjae Yu to Yonsei University

* Add Youngwoon Lee to Yonsei University

* Add Jinkyu Jeong to Yonsei University

* Add Kwanghyun Park to Yonsei University

* Move Yongjun Park to Yonsei University

abegel committed:
Add Andrew Begel to CMU (#6137)

noodle-lab committed:
Update new tenure-track faculties at University of Texas at Arlington (#6142)

* Update new tenure-track faculties at University of Texas at Arlington

* Update new tenure-track faculties at University of Texas at Arlington

* Update the homepage of Jiayi Meng at UTA

* Update the the homepages of new faculties at UTA

hosking committed:
Add Yun Kuen Cheung and Xiaoyu Sun, both now at ANU (#6144)

* Add Yun Kuen Cheung and Xiaoyu Sun, both now at ANU

* Update Steve Gould's GS identifier

* Correct Hongdong Li's GS identifier

* Anton disputes his removal

* Emeritus do not meet criteria

* Emeritus do not meet criteria

* Fix Anton's link (old one failed validity check)

* Revert to previous URL (not sure why it is failing the test)

* Flush Emeritus and Adjunts

* Aaron Quigley is now at CSIRO.

* Matthew Hole is 66% Math not CS.

* Drop Priscilla.

---------

Co-authored-by: Emery Berger <emery.berger@gmail.com>

tong1heng committed:
Add 10 faculty members at Renmin University of China (#6149)

* Add Cuiping Li at Renmin University of China

* Add Tong Li at Renmin University of China

* Add Bo Qin at Renmin University of China

* Add Jing Wang at Renmin University of China

* Add Yansong Zhang at Renmin University of China

* Add Kun He at Renmin University of China

* Update Cuiping Li's google scholar id

* Add Xun Liang at Renmin University of China

* Add Jie Zhao at Renmin University of China

* Add Haoqiong Bian at Renmin University of China

* Add Jinlong E at Renmin University of China

sameer-patil committed:
Removed faculty members who have left and added ones who joined. (#6152)

woriazzc committed:
Add Wei Zhang at East China Normal University (#6154)

ffmpbgrnn committed:
Add Linchao Zhu to Zhejiang University (#6155)

wniu9 committed:
Update the affiliation of new faculty of UGA - Jin Lu (#6157)

roberto-martinmartin committed:
Adding new faculty "Roberto Martín-Martín" to UT Austin CS (#6150)

* Adding new faculty "Roberto Martín-Martín" to UT Austin CS

* Only two names, the ones in DBLP

iombak committed:
added volkamer,rahmann, nagashima (#6146)

yaz91 committed:
Adding new hire at William&Mary (#6159)

* Add Yanfu Zhang to csrankings-y.csv

Yanfu Zhang, assistant prof at William&Mary starting in Aug 10

* Recent hire at WM - Yanfu Zhang

mlpacheco committed:
Adding Maria Leonor Pacheco to the CU Boulder faculty (#6160)

ryanpmcmahan committed:
Adding Jongouk Choi to UCF (#6162)

WeizheSyr committed:
Update Fanxin Kong (#6167)

Fanxin Kong transferred from Syracuse University to the University of Notre Dame.

emeryberger committed:
Removed non-TT faculty member.

emeryberger committed:
Updates.

chapering committed:
add 5 faculty members from WSU EECS (#6132)

* add 5 faculty members from WSU EECS

* removed career track faculty

bremen79 committed:
Updated Francesco Orabona (#6179)

liam0215 committed:
Removing Jonathan Mace, Keon Jang, and adding Laurent Bindschaedler to the Max Planck Society (#6180)

* Remove Jonathan Mace

Jonathan Mace left MPI-SWS to join Microsoft Research

* Remove Keon Jang

Keon Jang left MPI-SWS to join Rubrik as a software engineer

* Add Laurent Bindschaedler to Max Planck Society

stettberger committed:
Update for Hamburg University of Technology (#6182)

aronlaszka committed:
Update affiliation for Aron Laszka (moved from U of Houston to Penn State) (#6172)

Update affiliation for Aron Laszka (moved from University of Houston to Pennsylvania State University)
Reference for new affiliation: https://ist.psu.edu/directory/aql5923

xiaodong-yu committed:
Add a new hire at Stevens Institute of Technology (#6193)

xiaoningdu committed:
Add Xiaoning Du to Monash University (#6185)

* Update csrankings-x.csv

Add Xiaoning Du to Monash University

* disambiguation suffix added

weizi-li committed:
Add/change faculty of University of Tennessee (#6206)

vicenteor committed:
Add Maryam Aliakbarpour to the Rice University CS Faculty (#6218)

ZhijiaoXiao committed:
Add Zhijiao Xiao to Shenzhen University(Update csrankings-z.csv) (#6210)

* Update csrankings-z.csv

* Create csrankings-z.csv

* Update csrankings-z.csv

* Update csrankings-z.csv

update personal homepage

* Add Zhijiao Xiao to Shenzhen University(Update csrankings-z.csv)

* Delete Zhijiao Xiao from Shenzhen University(Update csrankings-z.csv)

* Add Zhijiao Xiao to Shenzhen University(Update csrankings-z.csv)

---------

Co-authored-by: Emery Berger <emery.berger@gmail.com>

xiaochaowei committed:
Change Chaowei Xiao's affilation in  csrankings-c.csv (#6219)

* Change chaowei Xiao's affilation in  csrankings-c.csv

* Update Chaowei Xiao's Affiliation at csrankings-c.csv

royzhz committed:
Adding Dawei Cheng to Tongji University (#6225)

kapravel committed:
Adding multiple faculty to NCSU (#6215)

* Adding Bowen Xu to NCSU

* Adding Wesley K. G. Assunção to NCSU

* Adding Dominik Wermke to NCSU

* Adding Chin Ho Lee to NCSU

* Adding Wujie Wen to NCSU

YichenZW committed:
Add Xiaoming Liu at Xi'an Jiaotong Univ. 3 (#6140)

* add Xiaoming Liu at Xi'an Jiaotong Univ.

* Add Xiaoming Liu at XJTU

---------

Co-authored-by: YichenW:) <84266501+ZachYC@users.noreply.github.com>

ufarooq committed:
Adding Umar Farooq to LSU (#6228)

* Update csrankings-u.csv for Umar Farooq

* Update csrankings-u.csv for Umar Farooq

yumeng5 committed:
Add Yu Meng to UVA (#6223)

MH7111997 committed:
Add Sumegha Garg, new Assistant Professor at Rutgers University (#6229)

christopherbatty committed:
Add Shlomi Steinberg at University of Waterloo (#6232)

rooshenas committed:
UIC Fall 2023 update (#6234)

* Added Anrin Chakraborti, Xiaoguang Wang, and Shweta Shweta to UIC as new faculty.
Moved Sidharth Kumar from University of Alabama to UIC.

Added DLBP alias for Pedram Rooshenas.

* Updated names based on DBLP

* Removed the added alias from dblp-aliases for Pedram Rooshenas.
Moved Amirmohammad Rooshenas from csrankings-a.csv to Pedram Rooshenas at csrankings-p.csv

---------

Co-authored-by: Pedram Rooshenas <pedram@Pedrams-MBP.attlocal.net>

yyk2020 committed:
Update csrankings-y.csv (#6235)

Shawn-Shan committed:
Update Bo Li and Ce Zhang's affiliation to University of Chicago (#6236)

Gofinge committed:
Add Hengshuang Zhao at The University of Hong Kong (#6244)

asch committed:
Move Changwoo Min from academia to industry (#6245)

helenamentis committed:
Update csrankings-h.csv (#6249)

Added Helena M. Mentis

praeclarumjj3 committed:
Update Prof. Humphrey Shi's Affiliation in csrankings-h.csv (#6250)

Co-authored-by: Emery Berger <emery.berger@gmail.com>

shili1986 committed:
Update csrankings-s.csv (#6253)

chenzhutian committed:
Update Niklas Elmqvist's affiliation to Aarhus (#6259)

* Update Niklas Elmqvist's affiliation to Aarhus

* update niklas's homepage

huseyinhisil committed:
Huseyin Hisil's inclusion CSRankings.com (#6262)

Huseyin Hisil has been with University of Wollongong since June 2023. This update includes his entry.

ckomus committed:
Ckomus patch 1 (#6275)

* Updated affiliation of Christian Komusiewicz

* Updated affiliation of Matthias Hagen

chenzhutian committed:
add new tenure-track faculties of UMN@CSE (#6257)

* add four new faculties of UMN@CSE

* add Stevie Chancellor to UMN@CSE

* Add Karthik Desingh to UMN@CSE

* add yao-yi chiang

* add Chang Ge to umn@cse

* add Ju Sun to UMN@CSE

* fix Chang Ge 0002

* Remove Ju Sun to pass the validation

ychen-stat-ml committed:
add 3 new APs for HKBU, and indicate Prof. Yike Guo moved to HKUST (#6261)

Co-authored-by: Freefighter <williamchen95eng@gmail.com>

muhaochen committed:
Adding Prof. Junshan Zhang to UC Davis (#6264)

dkqjrm committed:
add faculty of The Catholic University of Korea (#6265)

ZhaoweiTan committed:
Update csrankings-z.csv - Add Zhaowei to UCR (#6266)

saadhassan96 committed:
Updating Information: Saad Hassan (#6273)

* Added new faculty information

* Updated information for Saad Hassan

acbull committed:
add RPI professors (#6274)

* add RPI professors

* add Prof. Liu

* update

* update

* change https to http

suguman committed:
Adding Suguman Bansal from GaTech (#6279)

jxhe committed:
Update faculty affiliation (#6280)

xiaoleihuang committed:
Adding a new name (#6281)

leiyucs committed:
Update csrankings-l.csv (#6283)

updated Lei Yu homepage url

Junjie-Chu committed:
Stefan Bauer moves to TU Munich. Update the Google Scholar page and personal page. (#6294)

* Stefan Bauer moves to TU Munich

Stefan Bauer has moved to TU Munich. He has updated his home page.

* Stefan Bauer moves to TU Munich

* Anders Robertsson unfortunately passed away.

* Revert "Anders Robertsson unfortunately passed away."

This reverts commit 91e151f8259a455975614b4f2d3ad12679050320.

ShaiYer committed:
Update Moshe Babaioff, Amir Gilad, Sagie Benai, Liat Peterfreund,,Hebrew University (#6287)

* Update Hebrew University: 'Yossi Adi' 'Yossi Arjevani' 'Alon Eden' 'Tom Hope' 'David Hay' 'Yuval Peled'

* Update Moshe Babaioff,,Hebrew University

* Amir Gilad, Hebrew University

* Update Sagie Benaim, Hebrew University

* Update Liat Peterfreund,Hebrew University of Jerusalem

* Update SagieBenaim scholar ID csrankings-s.csv

montex committed:
Update csrankings-l.csv (#6298)

Adding entry for Leonardo Montecchi, full-time faulty at NTNU

clmiao committed:
Adding Clay Stevens and Liyi Li to Iowa State University CS Faculty (#6301)

* Update csrankings-c.csv

* Update csrankings-l.csv

kuldeepmeel committed:
Updating affiliation of K S Meel from NUS to University of Toronto (#6302)

ricpacca committed:
Add faculty to Carnegie Mellon University (#6288)

zaoxing committed:
Update the affiliation of Zaoxing Liu from BU to UMD (#6304)

Zaoxing Liu joined the CS department at University of Maryland - College Park.

torfah committed:
Add Hazem Torfah to csrankings-h.csv (#6308)

waehlisch committed:
add Markus Krötzsch and Roberto Calandra to TU Dresden (#6306)

yuanheTian committed:
Update the affiliation of Prof. Yan Song (#6312)

Update the affiliation of Prof. Yan Song

shuguang-52 committed:
Add Prof. Cairong Zhao to Tongji University (#6313)

cangiuli committed:
Add Carlo Angiuli to Indiana University. (#6316)

mbutrovich committed:
Update Jignesh M. Patel's affiliation (CMU), homepage, and Google Scholar ID. (#6317)

* Update Jignesh M. Patel's affiliation and homepage.

* Update Jignesh's Google Scholar ID.

chenzhutian committed:
Add Sun Ju to UMN (#6320)

use a separated PR in case cannot pass the validation

chitta66 committed:
Yanjie Fu - affiliation changed from UCF to Arizona State University (#6239)

Yanjie Fu moved from University of Central Florida to Arizona State University

chitta66 committed:
Hokeun Kim joined Arizona State University (#6240)

Hokeun Kim joined Arizona State University

hoonose committed:
Modify Waterloo Faculty (#6318)

* Add new Waterloo hires

* Removed Waterloo retirees

* Add cross-appointed faculty

* Update websites

* update Scholars

---------

Co-authored-by: Emery Berger <emery.berger@gmail.com>

plyfager committed:
Update csrankings-s.csv (#6324)

MohsenAnsari71 committed:
Adding a new faculty, affiliated to Sharif University of Technology (#6195)

* Adding a new faculty, affiliated to Sharif University of Technology

* updating the first name

mbutrovich committed:
Move Matei Zaharia to Berkeley (#6325)

earlence-uwm committed:
Add 14 new faculty to UCSD and move Alin Deustch from industry back to UCSD (#6310)

* add Lily Weng, Lianhui Qin (UCSD CSE)

* Prithiviraj Ammanabrolu (UCSD CSE) and Parinaz Naghizadeh (UCSD ECE)

* Yufei Ding (UCSD CSE), Yuanyuan Shi (UCSD ECE), Yang Zheng (UCSD ECE)

* Qipeng Liu (UCSD CSE)

* Alex Tamkin (UCSD CSE)

* Deepak Kumar (UCSD CSE)

* Fixed Yang Zheng -> Yang Zheng 0001

* Sam Lau and Soohyun Nam Liao (UCSD HDSI)

* Jun-Kun Wang (UCSD HDSI,ECE)

* Gal Mishne (UCSD HDSI)

* Alin Deutsch back to UCSD

* Alin Deutsch back to UCSD

* Fix Alin D. website

* Update csrankings-p.csv, removed Pritam Bhattacharya

---------

Co-authored-by: Emery Berger <emery.berger@gmail.com>

ricardor4 committed:
Added 2 new faculty members to Pontificia Universidad Catolica de Chile (#6327)

* added diego arroyuelo to PUC-Chile

* added miguel romero to PUC-Chile

samsonzhou committed:
Updated multiple faculty for Texas A&M University (#6333)

* Updated multiple faculty for Texas A&M University

* Update csrankings-t.csv

* Update csrankings-t.csv

mio-1q84 committed:
Update csrankings-l.csv (#6337)

Zjh-819 committed:
Add Songqiao Han, Wenting Tu, Yixin Tao and Yun Chen 0007 to SUFE (#6340)

* Add Yixin Tao and Yun Chen 0007 to SUFE

* Add Songqiao Han to SUFE

* Add Wenting Tu to SUFE

pandeydeep9 committed:
Update csrankings-q.csv (#6344)

Correction of website for Professor Qi Yu.

DiegoEliasCosta committed:
Updating Affiliation of Diego Costa (#6345)

diponomarev committed:
Adding three new faculty to Binghamton and removing one (Hui Lu moved to UT Arlington) (#6346)

* Update csrankings-d.csv

Added Dali Ismail to Binghamton University

* Update csrankings-y.csv

Adding Yincheng Jin to Binghamton University

* Update csrankings-z.csv

Adding Zhen Xie to Binghamton University

* Update csrankings-h.csv

Removing Hui Lu from Binghamton University

eduardoblanco committed:
Update Ellen Riloff's affiliation and website (#6351)

Utah -> Arizona
Announcement: https://www.cs.arizona.edu/news/ellen-riloff-joining-ua-department-computer-science

cdstanford committed:
UC Davis updates: new faculty, move some to emeritus (#6353)

* Update csrankings-a.csv

* Update csrankings-b.csv

* Update csrankings-d.csv

* Update csrankings-a.csv

* Update csrankings-d.csv

* Update csrankings-i.csv

* Update csrankings-k.csv

* Update csrankings-m.csv

* Update csrankings-n.csv

* Update csrankings-p.csv

* Update csrankings-r.csv

* Update csrankings-s.csv

* Update industry.csv

* Update csrankings-s.csv

* Update emeritus.csv

anandpiyer committed:
Update csrankings-a.csv (#6355)

hongxin001 committed:
add Hongxin Wei @ SUSTech, new assistant professor (#6356)

* add Hongxin Wei

* add Hongxin Wei

rabinadk1 committed:
Update Xio Wang, Samir Khuller, and Bryan Pardo's homepages (#6358)

* Update Xio Wang's website

* Update Samir Khuller's homepage

* Update Bryan Pardo's homepage

cooelf committed:
Update csrankings-z.csv (#6357)

pandeydeep9 committed:
Update using correct links for Professor Qi Yu (#6359)

* Update csrankings-q.csv

Correction of website for Professor Qi Yu.

* Correct link for Professor Qi Yu (Both website and google scholar id)

---------

Co-authored-by: Emery Berger <emery.berger@gmail.com>

Commit:
Update csrankings-a.csv (#6365)

portokalidis committed:
Update affiliation (#6367)

https://software.imdea.org/people/georgios.portokalidis/

grobalex committed:
Updated various Northeastern Faculty  (#6352)

* Added Ada Lerner

* Added Robin Walters

* Added Silvio Amir

* Adding Michael Everett

* Added Megan Hofmann

* Added multiple Khoury professors

* Updated Long Lu's Google Scholar ID

* Updated Cheng Tan's DBLP ID to remove unwanted character

* Fixed affiliation typo for Robin Walters

* Updated Kevin Fu's record

* Delete .DS_Store

renevidal committed:
Change Rene Vidal's affiliation to University of Pennsylvania from Johns Hopkins in csrankings-r.csv (#6369)

Rene Vidal moved to Upenn on January 1, 2023
https://penntoday.upenn.edu/news/rene-vidal-appointed-penn-integrates-knowledge-university-professor-penn

mabdula committed:
Hubie Chen moved to KCL (#6370)

* Hubie Chen moved to KCL

* Fixed missing field

vvv214 committed:
add new faculty to University of Virginia (#6371)

* Update csrankings-c.csv

add Chang Lou to University of Virginia

* Ferdinando Fioretto moved to University of Virginia

* Wei-Kai Lin joined University of Virginia

* Yen-Ling Kuo joined University of Virginia

* Chen-Yu Wei joined University of Virginia

* Hyojoon Kim joined University of Virginia

* Chen-Yu Wei joined University of Virginia

* Yu Meng joined University of Virginia

* Matheus Ferreira joined University of Virginia

* Zezhou Cheng joined University of Virginia

* Kun Qian joined University of Virginia

* Amanda Watson joined University of Virginia

* Tom Hartvigsen joined University of Virginia

* add Michael Albert to University of Virginia

---------

Co-authored-by: Emery Berger <emery.berger@gmail.com>

gsmaragd committed:
Alexios Voulimeneas joined TU Delft (#6372)

gsmaragd committed:
Evangelia Markatou joined  TU Delft (#6373)

phananh1010 committed:
update a faculty at George Mason University, Applied Information Technology (#6349)

gabrer committed:
Update csrankings-g.csv (#6376)

Added G. Pergola

ZexinLi0w0 committed:
Update Cong Liu's Homepage and Google Scholar (#6377)

mabdula committed:
Added Yali Du to KCL (#6383)

* Added Yali Du to KCL

* Fixed name by additing number on dblp

stefan0711 committed:
Add a new faculty to Florida International University (#6384)

tagbenja committed:
Added Faculty Member at Monash University (#6386)

* added tenured faculty member at Monash University

added Benjamin Tag, tenured faculty member, in Faculty of IT, Monash University

* Revert "added tenured faculty member at Monash University"

This reverts commit 9b0862c7c98d5de9f290b290dc9ac0b394b17a61.

* Added Faculty Member at Monash University

# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (please don't contact us for at least three months for updates, unless you are a sponsor - see below).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR PULL REQUEST WILL BE SUMMARILY REJECTED.
You must read and check **all** the boxes below by filling them in with an X or your PR will be rejected.

**The Basics**

- [x] All pull requests and issues must come from non-anonymous accounts. Make sure your GitHub profile contains your full name.

- [x] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [x] Combine multiple updates to a single institution into a **single PR.**

- [x] Only submit one pull request per institution.

- [x] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [x] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [x] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [x] Check to make sure that you have no spaces after commas, or any missing fields.

- [x] Check to make sure the home page is correct.

- [x] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [x] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [x] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [x] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone**

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

* erased space after comma

- erased space after comma
- newly added faculty at Monash University

emeryberger committed:
Added missing dependency.

heades committed:
Fall 2023 Augusta University additions/removals. (#6389)

yang-sec committed:
Add two CS faculty members to University of Kentucky (#6393)

* Update csrankings-y.csv

add a faculty member to University of Kentucky

* Update csrankings-h.csv

add a faculty member to University of Kentucky

osalbahr commented:
Ok.... I tried to rebase but I guess I did it wrong. I will just redo this PR.
